src/mon/OSDMonitor.cc

```diff
@@ -93,6 +95,164 @@ const uint32_t MAX_POOL_APPLICATION_LENGTH = 128;
 
 } // anonymous namespace
 
+
+namespace {
+
+enum class flag_result_t : int { not_verified = -EPERM, not_set = 0, ok = 1 };
+
```

Comment:
In the next line, you'll see what's up with the "command_projection" business. There is reasoning behind this, and I understand that it might be odd at first. Please ask me questions!